UICatalog/UICatalog.cs

```diff
@@ -270,6 +272,21 @@ public UICatalogTopLevel ()
 				ScenarioListView.SelectedItem = _cachedScenarioIndex;
 			}
 
+			private void SetTitlePaddingToAlignScenariosHeader ()
+			{ 
+				// Pos.Anchor is internal so we have to use reflection to access it
+				var anchor = typeof (Pos).GetMethod ("Anchor", BindingFlags.Instance | BindingFlags.NonPublic);
+				var splitterWidth = (int)anchor.Invoke (
+					SplitContainer.SplitterDistance, new object [] { SplitContainer.Bounds.Width}
+					);
+
+				var newTitle = $"Categories ({ContentPane.ShortcutTag})";
+				newTitle = newTitle.PadRight (splitterWidth + 1, (char)Driver.HLine);
+				newTitle += "Scenarios";
```

Comment:
> Why can't we have a "T" here?
> 
> <img alt="image" width="212" src="https://user-images.githubusercontent.com/585482/210138272-19659801-2282-4420-aef7-67ecff03d45b.png">

Should be able to do that with a bit of work.

Currently a `SplitContainer` control is 2 borderless `View` panels and a horizontal line down the middle.

The `Title` (and border) come from a `FrameView`.  I align the Title using this approach here which feels a bit ad-hoc.  I could update this method to have the `Title` use a T at the right horizontal position so that it aligns.  But there would be now corresponding upside down T at the bottom since those Runes come from the `Title`.

Alternatively I could look into providing a version of `SplitContainer` that draws its own (optionally captioned  border).